Translations/de.lproj/Localizable.strings

```diff
@@ -217,18 +217,18 @@ Note that the short string is limited to 30 characters and the long string is li
 "Monthly sip of coffee" = "Monthly sip of coffee";
 "Gift Keka a sip of coffee each month" = "Gift Keka a sip of coffee each month";
 "Monthly coffee" = "Monthly coffee";
```

Comment:
Can you translate those too?

Reply:
Sure, I have translate those in the new commit